docs/backends/HPC.md

```diff
@@ -18,11 +18,20 @@ Cromwell is configured with a root execution directory which is set in the confi
 When Cromwell runs a workflow, it first creates a directory `<cromwell_root>/<workflow_uuid>`.  This is called the `workflow_root` and it is the root directory for all activity in this workflow.
 
 Each `call` has its own subdirectory located at `<workflow_root>/call-<call_name>`.  This is the `<call_dir>`.
-Any input files to a call need to be localized into the `<call_dir>/inputs` directory. There are different localization strategies that Cromwell will try until one works. Below is the default order specified in `reference.conf` but it can be overridden:
+Any input files to a call need to be localized into the `<call_dir>/inputs` directory. There are different localization strategies that Cromwell will try until one works:
 
 * `hard-link` - This will create a hard link to the file
 * `soft-link` - Create a symbolic link to the file. This strategy is not applicable for tasks which specify a Docker image and will be ignored.
 * `copy` - Make a copy the file
+* `cached-copy` introduced in cromwell 41 as an experimental feature. This copies files to a file cache in 
```

Comment:
Version number needs updating (or removing, IMO people can find the version from the changelog)

Reply:
Yes please remove the version number.